.github/ISSUE_TEMPLATE/package_build_request.yml

```diff
@@ -0,0 +1,105 @@
+          - Review and comply with our [Anaconda Community Code of Conduct](https://www.anaconda.com/community-code-of-conduct/).
+
+        Thank you for your contribution and for helping Anaconda better serve its community of users. 🐍
+
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: Please confirm and check all the following options.
+      options:
+        - label: I added a descriptive title
+          required: true
+        - label: I searched for other package requests and couldn't find a duplication
+          required: true
+
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: "Please confirm the following:"
+      options:
+        - label: I added a descriptive title (including the requested package **name**, **version**, and **platform**).
+          required: true
+        - label: I searched the [anaconda](https://anaconda.org/anaconda) channel and did not find the package I'm requesting.
+          required: true
+        - label: I searched through [existing issues](https://github.com/ContinuumIO/anaconda-issues/issues) and did not find a duplicate request for this package.
+          required: true
+
+  - type: checkboxes
+    id: package-request-type
+    attributes:
+      label: Request type
+      description: Is this a new, update, or rebuild request?
+      options:
+        - label: New (i.e. no versions of this package currently exist in the "anaconda" channel)
+        - label: Update (i.e. older versions of this package exist and I'm requesting an updated version)
+        - label: Rebuild (i.e. something is wrong with an existing version of this package and needs to be rebuilt)
+
+  - type: input
+    id: package-name
+    attributes:
+      label: Name
+      description: Name of the package.
+      placeholder: ex. numpy
+    validations:
+      required: true
+
+  - type: input
+    id: package-version
+    attributes:
+      label: Version
+      description: Version of the package you're requesting.
+      placeholder: ex. 1.20.0
+    validations:
+      required: true
+
+  - type: input
+    id: package-platform
+    attributes:
+      label: Platform(s)
+      description: Platform(s) you need the package to be built for.
+      placeholder: ex. linux-64
+    validations:
+      required: true
+
+  - type: input
+    id: package-source-code
+    attributes:
+      label: Location of source code
+      description: Where the source code can be found for this project (e.g. the project's Github repo or PyPI page).
+    validations:
+      required: false
+
+  - type: input
+    id: package-conda-forge-feedstock
+    attributes:
+      label: Conda-Forge feedstock
+      description: Link to the Conda-Forge feedstock (if applicable)
+    validations:
+      required: false
+
+  - type: textarea
+    id: reason
+    attributes:
+      label: (OPTIONAL) Reason for this request
```

Comment:
I feel like this one shouldn't be optional.  This is essentially "why is this important"   For instance, if someone is reporting an issue with a package that needs to be rebuilt - having in the why section that the package being broken means a whole class of packages doesn't work and it is a major issue - is different than if someone has a pet project that almost no one has heard of that they think would be cool to see in the repo.

Reply:
Done. Great point. Thank you!